ProjectManager/ProjectManager/ViewModel/CellViewModel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CellViewModel.swift
+//  ProjectManager
+//
+//  Created by Kyo on 2023/01/15.
+//
+
+import Foundation
+
+final class CellViewModel {
+    private var data: Todo {
+        didSet {
+            dataHandler?(data)
+        }
+    }
+    
+    private var dataHandler: ((Todo?) -> Void)?
+    
+    init(data: Todo) {
+        self.data = data
+    }
+}
+
+// MARK: - Method
+extension CellViewModel {
+    func bindDate(handler: @escaping (Todo?) -> Void) {
+        handler(data)
+        self.dataHandler = handler
+    }
+    
+    func checkOverDeadLine() -> Bool {
+        if data.isOverDeadLine {
+            return true
+        }
+        return false
```

Comment:
if문의 결과로 Bool을 반환하는 경우는 if문 컨디션을 그대로 리턴해도 좋겠습니다.
```suggestion
        return data.isOverDeadLine
```

Reply:
수정했습니다! :)